improver_tests/psychrometric_calculations/test_PhaseChangeLevel.py

```diff
@@ -432,16 +417,6 @@ def test_null_lat_lon(self):
         result = plugin.find_max_in_nbhood_orography(cube)
         self.assertArrayAlmostEqual(result.data, expected_data)
 
-    def test_error_lat_lon(self):
```

Comment:
Can you add a test, perhaps in test_OccurrenceInVicinity, to show that it works for both equal-area and lat-lon grids now? At least if you specify the right radius.

Reply:
I've tested the use of a zero radius / grid point radius with a latlon grid, and a non-zero grid point radius with a latlon grid. A non-zero radius with a latlon grid will lead to an exception when the code attempts to calculate a number of grid points; this is actually raised in iris as it cannot convert degrees into metres (with metres enforced in the improver call). I'm happy that this will fail, so prevents silly usage and was leaving it at that. but we could improve the exception message if you think it's worth it?

Reply:
No. I don't think we need to trap and replace the Iris exception message. We could end up masking some other exception.